ExtendCanvas/src/main/java/com/microsoft/device/display/samples/extendcanvas/ExtendedCanvasActivity.kt

```diff
@@ -3,23 +3,36 @@
  * Licensed under the MIT License.
  *   
  */
+
 package com.microsoft.device.display.samples.extendcanvas
 
+import android.annotation.SuppressLint
 import android.os.Bundle
 import android.webkit.WebView
 import androidx.appcompat.app.AppCompatActivity
 import com.microsoft.device.display.samples.extend.R
 
-class MainActivity : AppCompatActivity() {
+class ExtendedCanvasActivity : AppCompatActivity() {
+    companion object {
+        const val GOOGLE_MAPS_URL = "https://www.google.com/maps/@?api=1&map_action=map&center=47.652822, -122.134596&zoom=12"
+    }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.activity_extended_canvas)
+        setupActionBar()
+        setupWebView()
+    }
+
+    private fun setupActionBar() {
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
         supportActionBar?.setHomeAsUpIndicator(R.drawable.menu)
```

Comment:
This hamburger menu item doesn't open any menu, wouldn't it be better to remove it in order to prevent confusion?

Reply:
Done